internal/stack/dump.go

```diff
@@ -18,8 +18,6 @@ const (
 	fleetServerService  = "fleet-server"
 )
 
-var observedServices = []string{"elasticsearch", elasticAgentService, fleetServerService, "kibana", "package-registry"}
-
```

Comment:
Now dump and status would be getting the services directly from the docker daemon.

These containers are obtained from here:
https://github.com/elastic/elastic-package/pull/1374/files#diff-b4efb1ac558d15d3077525cb9be59dce12ee723a4749253e9ca7cc860419f1f8R401-R462